app/pages/project/home/project-home.jsx

```diff
@@ -69,7 +73,7 @@ const ProjectHomePage = (props) => {
 
             <div>
               <img role="presentation" src={avatarSrc} />
-              <span>&quot;{props.project.researcher_quote}&quot;</span>
+              <span>&quot;{translations.strings.project.researcher_quote || props.project.researcher_quote}&quot;</span>
```

Comment:
This should be updated to use the translation wrapper pattern that's being used by workflow tasks.